debug/api.go

```diff
@@ -52,43 +48,6 @@ type HandlerT struct {
 	traceFile string
 }
 
-// Verbosity sets the log verbosity ceiling. The verbosity of individual packages
-// and source files can be raised using Vmodule.
-func (*HandlerT) Verbosity(level int) {
-	glogger.Verbosity(slog.Level(level))
-}
-
-// Vmodule sets the log verbosity pattern. See package log for details on the
-// pattern syntax.
-func (*HandlerT) Vmodule(pattern string) error {
-	return glogger.Vmodule(pattern)
-}
-
-// MemStats returns detailed runtime memory statistics.
-func (*HandlerT) MemStats() *runtime.MemStats {
-	s := new(runtime.MemStats)
-	runtime.ReadMemStats(s)
-	return s
-}
-
-// GcStats returns GC statistics.
-func (*HandlerT) GcStats() *debug.GCStats {
-	s := new(debug.GCStats)
-	debug.ReadGCStats(s)
-	return s
-}
-
-// CpuProfile turns on CPU profiling for nsec seconds and writes
-// profile data to file.
-func (h *HandlerT) CpuProfile(file string, nsec uint) error {
-	if err := h.StartCPUProfile(file); err != nil {
-		return err
-	}
-	time.Sleep(time.Duration(nsec) * time.Second)
-	h.StopCPUProfile()
-	return nil
-}
```

Comment:
I confirm removing this **does not remove** possibility to use following `sonictool cli` features: :+1: 
* `debug.memStats()`
* `debug.gcStats()`
* `debug.cpuProfile('test',10)`
* `debug.goTrace('test', 10)`
* `debug.blockProfile('test', 5)`
* etc... - all this is handled by HandlerT in go-ethereum-sonic/internal/debug